CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## ?.?.?
```

Comment:
Dato istedet for ? tegn?

Reply:
Den blir fylt ut av `bump2version` når man lager neste utgivelse. Pleier å la det stå `?.?.?` her siden vi ikke vet når utgivelsesdatoen blir før selve utgivelsen lages. :)